antora-ui-camel/src/css/header.css

```
@@ -412,9 +462,22 @@ body {
   padding-top: 0.3rem;
 }
 
+#search_results div.footer-search {
+  display: inline-flex;
+  float: right;
+  margin-right: 0.75rem;
```

`float` doesn't make sense in flex layout, better to use padding vs margin so not to introduce scrollbars

```suggestion
  padding: .75rem;
  width: 100%;
```